src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
Its shared source, so it doesn't actually build here, it essentially gets copied into the user assembly and builds there. Kinda wonky I know.

Sent from my Windows Phone
________________________________
From: Doug Bunting<mailto:notifications@github.com>
Sent: ý2/ý17/ý2015 6:23 PM
To: aspnet/Common<mailto:Common@noreply.github.com>
Cc: Taylor Mullen<mailto:Taylor.Mullen@microsoft.com>
Subject: Re: [Common] Add MVC common utilities. (#1)


In src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json<https://github.com/aspnet/Common/pull/1#discussion_r24875025>:

> @@ -0,0 +1,15 @@
> +{
> +    "version": "1.0.0-*",
> +    "shared": "*.cs",
> +    "dependencies": {


I was actually wondering about the opposite issue: how does kpm pack or kpm build in this directory succeed without the dependency?

—
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Common/pull/1/files#r24875025>.